R/build-search-docs.R

```diff
@@ -88,7 +88,7 @@ url_node <- function(url) {
 #'
 #' @inheritParams build_articles
 #' @export
-#'
+#' @seealso `vignette("search")`
```

Comment:
There's little enough in that vignette, I think I'd almost prefer going in the opposite direction — just point to `?build_search` from the vignette. (And then maybe remove the vignette from the main navbar since it will only apply to BS3. Or maybe even remove it altogether with a redirect since BS3 is deprecated?)

Reply:
Sounds good, I will revert this change and I can open a new issue so that you can make the final decision.